content/tracing/setup/go.md

```diff
@@ -80,53 +100,60 @@ ___
 | SQL                 | https://godoc.org/github.com/DataDog/dd-trace-go/contrib/database/sql | https://godoc.org/github.com/DataDog/dd-trace-go/contrib/database/sql                      |
 | SQLx                | https://github.com/jmoiron/sqlx                                       | https://godoc.org/github.com/DataDog/dd-trace-go/contrib/jmoiron/sqlx                      |
 
-### OpenTracing API
+___
+
+Make sure to take a look at our integrations package [godoc page][contrib godoc] for an in-depth look.
 
-Datadog APM client that implements an [OpenTracing][3] Tracer.
+## OpenTracing Support
 
-**Initialization**
+Import the [`opentracer` package][opentracing godoc] to expose the Datadog tracer as an [OpenTracing][3] compatible tracer.
 
-To start using the Datadog Tracer with the OpenTracing API, you should first initialize the tracer with a proper `Configuration` object:
+### Example
+
+A basic usage would be:
 
 ```go
+package main
+
 import (
-  // ddtrace namespace is suggested
-  ddtrace "github.com/DataDog/dd-trace-go/opentracing"
-  opentracing "github.com/opentracing/opentracing-go"
+    "github.com/opentracing/opentracing-go"
+
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/opentracer"
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
 func main() {
-  // create a Tracer configuration
-  config := ddtrace.NewConfiguration()
-  config.ServiceName = "api-intake"
-  config.AgentHostname = "ddagent.consul.local"
-
-  // initialize a Tracer and ensure a graceful shutdown
-  // using the `closer.Close()`
-  tracer, closer, err := ddtrace.NewTracer(config)
-  if err != nil {
-    // handle the configuration error
-  }
-  defer closer.Close()
-
-  // set the Datadog tracer as a GlobalTracer
-  opentracing.SetGlobalTracer(tracer)
-  startWebServer()
-}
-```
+    // Start the regular tracer and return it as an opentracing.Tracer. You may use
```

Comment:
OpenTracing Tracer?

Reply:
Not sure which is more appropriate in this case. The "_english_" version or the technical version. `opentracing.Tracer` is the interface expected to be returned by the implementations.

Reply:
we're in the code-land, so I'd say to keep `opentracing.Tracer`

Reply:
Done.